presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -193,12 +192,15 @@ public ConnectorOutputTableHandle beginCreateTable(ConnectorSession session, Con
                     SPLIT_COUNT_PROPERTY, PAGES_PER_SPLIT_PROPERTY, ROWS_PER_PAGE_PROPERTY));
         }
 
+        Duration pageProcessingDelay = (Duration) tableMetadata.getProperties().get(PAGE_PROCESSING_DELAY);
+
         return new BlackHoleOutputTableHandle(new BlackHoleTableHandle(
```

Comment:
Now that the output handle has two arguments, introduce a variable for the table handle